.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python: -"3.6"
```

Comment:
Let's use 3.7

Reply:
should there be any change is `Pipfile` , where  it is [requires]python_version="3.6"

Reply:
Thanks.